libc/src/unistd/linux/lseek.cpp

```diff
@@ -14,8 +14,8 @@
 #include "src/__support/OSUtil/syscall.h" // For internal syscall function.
 #include "src/__support/common.h"
 
-#include <sys/syscall.h> // For syscall numbers.
-#include <unistd.h>      // For off_t.
+#include "hdr/unistd_macros.h" // For off_t.
```

Comment:
if this is for `off_t` it may need the correct type header instead of `unistd_macros.h`